src/WebJobs.Script/Dispatch/FunctionDispatcher.cs

```diff
@@ -36,13 +36,13 @@ internal class FunctionDispatcher : IFunctionDispatcher
             IRpcServer server,
             CreateChannel channelFactory,
             TraceWriter traceWriter,
-            List<WorkerConfig> workers)
+            IEnumerable<WorkerConfig> workers)
         {
             _eventManager = manager;
             _server = server;
             _channelFactory = channelFactory;
             _traceWriter = traceWriter;
-            _workerConfigs = workers ?? new List<WorkerConfig>();
+            _workerConfigs = workers.ToList() ?? new List<WorkerConfig>();
```

Comment:
`ToList()` shouldn't return `null`, and if `workers` is `null` here, this would throw.... did you mean to have `workers?.ToList()`?

Reply:
yep, good catch